src/Resources/views/includes/sidebar/_profile.html.twig

```diff
@@ -36,14 +36,14 @@
     >
         <div class="flex flex-col gap-y-2 m-4 -mb-1 px-4 pt-2 pb-3 rounded-sm shadow-md z-10 bg-white ring-1 ring-black ring-opacity-5 focus:outline-none">
             <div class="border-b">
```

Comment:
Nit: Should be a HTMLParagraphElement for the content to have correct semantic meaning.

Reply:
@renestalder I guess you mean the `span` element which includes the username?

Reply:
@renestalder Okay, understand. So you meant the div with the class `border-b`. I don't see exactly which line you referenced to.

But we should avoid wrapping divs within p-tags, which would be the case then.
Maybe convert the divs into spans. But we still have a markup within the p tag which isn't intended to be for that, right?

Reply:
@marcwieland95 The information, what line is commented, is visible in the code preview:

![Screenshot 2024-07-22 at 13 16 24](https://github.com/user-attachments/assets/d3997c23-a079-41fb-87c1-faa1b0dbe03d)

-36 + 36 means line 36.

> But we should avoid wrapping divs within p-tags, which would be the case then.

That's true, but the whole markup is problematic because it's used for styling purpose instead of semantic purpose. The `<img>` doesn't need a div wrapper. And if so, that could be a `<picture>` element then. But I don't see a good reason for it to have a wrapper, could just be on the img itself.

Reply:
@renestalder About the element. I'm sorry, but I still don't get it. Line 36 is the closing tag which for sure isn't what you mean.

The wrapper around is absolutely for styling. I don't know how you would arrange the elements accordingly other then turning everything into a grid container.

Reply:
@marcwieland95 aah, now I see. Totally my fault.

So, the possibilities would be either:

p = Logged in as
p = wrapper
  img = Profile Picture
  span = username

Or without the additional wrapper:

p = wrapper
  span = Logged in as
  img = Profile Picture
  span = username

The second option can be achieved with Flex, Flex Wrap and a static Flex Base for the Image (Flex-Growing set to zero).

Also would consider the second option better semantically.

Reply:
Got it.
Worked and is for sure better, yes.

![CleanShot 2024-07-22 at 15 38 24@2x](https://github.com/user-attachments/assets/026eb577-3a98-4ea6-9288-60dc40c1f37e)